src/Sarif/Readers/DelegatingStream.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    internal class DelegatingStream : Stream
```

Comment:
>DelegatingStream [](http://example.com/codeflow?start=19&length=16)

Rename this type NonDisposingDelegatingStream.

Reply:
renamed